### PR TITLE
Does my coding partner follow JS best practices?

### PR DESCRIPTION
**Pull request by amon-cofie**
This is my pull request for peer review. I made a copy of my code from my to-do-list code for this review

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  margin-top: 3rem;
+}
+
+.make-flex {
+  display: flex;
+  justify-content: space-between;
+  height: 3rem;
+  align-items: center;
+  padding-left: 1rem;
+  padding-right: 1rem;
```
Comment (vflorin1995) [inline src/style.css:23]: Please avoid DRY. Instead, you can make one class with those two proprieties and apply it in every place you use them.

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+.to-do-list-app-container {
+  border: 5px solid black;
+  width: 60vw;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 3rem;
+}
+
+.make-flex {
+  display: flex;
```
Comment (vflorin1995) [inline src/style.css:18]: Please avoid DRY. Instead, you can make one class with those two proprieties and apply it in every place you use them.

**Review by vflorin1995 — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 1 above)
Comment (amon-cofie) [reply src/style.css]: Thank you for pointing that out. I will make the changes

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  transition: 0.5s;
+}
+
+.to-do-list-app-container {
+  border: 5px solid black;
+  width: 60vw;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 3rem;
+}
```
Comment (NahnahAJ) [inline src/style.css:15]: Please rewrite this code

**Pull request merged**